recipes/satsmooth/meta.yaml

```diff
@@ -0,0 +1,65 @@
+
+{% set name = "satsmooth" %}
+{% set version = "1.5.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jgrss/satsmooth/archive/refs/tags/v{{ version }}.zip
+  sha256: 344ea9558e6d6f73b6fedf60a47fdda0fe805b2b160feb2cb38e99fd8a6d03be
+
+# build:
+#   noarch: python
+#   script: {{ PYTHON }} -m pip install . -vv
+#   number: 0
+
+
+build:
+  noarch: python
+  script:
+    - python setup.py build
+    - python setup.py install
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }} 
+    - {{ compiler('cxx') }}  # Add this line for C++ compilation
+
+  host:
+    - python >=3.8, <3.10
+    - cython >=0.29.0,<3.0.0
+    - numpy
```

Comment:
It looks like you are compiling against numpy in some of your c-extensions. See:
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy